psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -1845,10 +1851,12 @@ void HF::print_energies()
 
     Process::environment.globals["SCF ITERATIONS"] = iteration_;
 
+#ifdef USING_libefp
     // Only print this alert if we are actually doing EFP or PCM
     if(pcm_enabled_ || ( Process::environment.get_efp()->get_frag_count() > 0 ) ) {
         outfile->Printf("    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.");
     }
+#endif
```

Comment:
Good point. I released the print message from ifdef, as `get_efp` and `get_frag_count` are safe to use, libefp enabled or not.